File: Geek-Start-Editorials/CSE-QUE/While-Loop 3/Solution.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) { 
        int n,j,i;
    Scanner s = new Scanner(System.in);
    n = s.nextInt(); i = s.nextInt();
        j = s.nextInt();
    int flag = 0;
    for(++i;i<j;i++){
        if(i%n==0) {
            System.out.print(i+" ");
            flag= 1;
        }
    }
    if(flag==0) System.out.println("No");
   
    }
}

File: Geek-Start-Editorials/CSE-QUE/While-Loop 3/Solution.c
#include <stdio.h>

int main() {

    int n,j,i,flag = 0;;

    scanf("%d",&n);
    scanf("%d",&i);
    scanf("%d",&j);
    
    for(++i;i<j;i++){
        if(i%n==0) {
            printf("%d ",i);
            
            flag= 1;
        }
    }
    if(flag==0)
        printf("No");
    return 0;
    
}
